src/scripts/components/surah/AyahsList.js

```diff
@@ -13,14 +14,19 @@ class AyahsList extends React.Component {
 
   list() {
     if (this.props.ayahs.length === 0) {
-        return <Loader />;
+      return <Loader />;
     }
 
-    return this.props.ayahs.map(ayah => {
-        return <Ayah ayah={ayah}
-                     key={`${ayah.surah}-${ayah.ayah}-ayah`}
-                     readingMode={this.props.isReadingMode} />;
-    });
+    if (this.props.isReadingMode) {
+      return this.props.lines.map((line, index) => {
+        return <Line line={line} key={`${index}-line`} />;
+      });
+    }
+    else {
+      return this.props.ayahs.map(ayah => {
+        return <Ayah ayah={ayah} key={`${ayah.surah}-${ayah.ayah}-ayah`} />;
+      });
```

Comment:
Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.